test/task-with-invalid-credentials.json

```diff
@@ -0,0 +1,13 @@
+{
+  "StartAt": "p",
+  "States": {
+    "p": {
+      "Type": "Task",
+      "Resource": "foo:bar",
+      "Credentials": {
+        "rollArn": "fizz:buzz"
+      },
```

Comment:
The ASL spec does not require any specific fields for `Credentials`, this test can just verify the wrong type (e.g. a String instead of Object) is invalidated.